src/docs/pages/guides/troubleshooting/popped-out-element-is-rendered-in-wrong-container.md

```diff
@@ -77,14 +72,14 @@ export default function App() {
       </LdModal>
       <LdButton
         onClick={() => {
-          setOpen(true);
+          setOpen(true)
         }}
       >
         Open Modal
       </LdButton>
     </>
-  );
+  )
 }
 ```
 
-<docs-page-nav prev-href="guides/sandbox-applications/" next-title="FAQ" next-href="guides/faq/"></docs-page-nav>
+<docs-page-nav prev-href="guides/sandbox-applications/" next-title="FAQ" next-href="guides/faq/"></docs-page-nav>
```

Comment:
Since this is now a sub page of the trouble shooting guide, I would remove the page nav here.
```suggestion
```

Reply:
I was expecting to add some additional troubleshooting guides (over time ;)) so once there are more this next button makes more sense, right. Although I think it's also not too bad / wrong to have a next step here leading to the FAQ which would be the next page if you are reading the guides page by page. 

The more I think about it, the more I'd prefer to have something at the end of the page. At least going back to the overview. 🤔

Reply:
Well, it‘s like entering building A, taking the elevator up to the third floor and then exiting through the window for building B. 😅 I don’t know. You decide! 🙈